kpops/components/base_components/base_defaults_component.py

```diff
@@ -128,25 +127,26 @@ def extend_with_defaults(cls, config: KpopsConfig, **kwargs: Any) -> dict[str, A
             elif is_dataclass(v):
                 kwargs[k] = asdict(v)
 
-        log.debug(
-            typer.style(
-                "Enriching component of type ", fg=typer.colors.GREEN, bold=False
-            )
-            + typer.style(cls.type, fg=typer.colors.GREEN, bold=True, underline=True)
+        env_pipeline_path = ENV.get("pipeline_path")
+        if not env_pipeline_path:
+            env_pipeline_path = Path()
+        pipeline_path = Path(env_pipeline_path)
```

Comment:
possible to shorten this a bit?
```suggestion
        pipeline_path = Path(ENV.get("pipeline_path", ""))
```